src/components/search/components/searchMenu/filters/MultiSelectFilter.tsx

```diff
@@ -81,11 +81,13 @@ const MultiSelectFilter = (props: Props) => {
 
     /* OnChange of Search Query: refresh aggregations by query */
     useEffect(() => {
-        const copyAggregations = { ...aggregatons };
+        const copyAggregations = { ...aggregations };
 
         /* Function to Refresh Aggregations */
         const RefreshAggregations = () => {
             GetSpecimenAggregations(searchFilters).then((aggregations) => {
+                console.log(aggregations);
```

Comment:
Couple of log statements. Do want to keep those or were they only for testing?

Reply:
Good catch